phy/cluster/manual/store.py

```diff
@@ -308,9 +227,25 @@ def _make_func(name):
         # Register the StoreItem instance.
         self._items.append(item)
 
+    def load(self, name, clusters, spikes):
+        assert _is_array_like(clusters)
+        load = getattr(self, name)
+
+        # Concatenation of arrays for all clusters.
+        arrays = np.concatenate([load(cluster) for cluster in clusters])
+        # Concatenation of spike indices for all clusters.
+        spike_clusters = np.concatenate([self._spikes_per_cluster[cluster]
+                                         for cluster in clusters])
+        assert np.all(np.in1d(spikes, spike_clusters))
+        idx = _index_of(spikes, spike_clusters)
+        return arrays[idx, ...]
+
     def update(self, up):
+        # TODO: update self._spikes_per_cluster
         # Delete the deleted clusters from the store.
-        self._store.delete(up.deleted)
+        self._memory.delete(up.deleted)
+        self._disk.delete(up.deleted)
```

Comment:
yeah, this will delete everything too. we should really test `delete` and check that only that file is deleted correctly. also we should test by randomly adding a few bits to one file and checking the whole thing is correctly deleted and regenerated. This is the kind of thing which might be weird cross-platform...